python/cudf/cudf/tests/dataframe/methods/test_reductions.py

```diff
@@ -318,9 +318,19 @@ def test_dataframe_axis1_unsupported_ops(op):
         "any",
     ],
 )
-def test_dataframe_reductions(data, axis, func, skipna):
+def test_dataframe_reductions(request, data, axis, func, skipna):
     pdf = pd.DataFrame(data=data)
-    gdf = cudf.DataFrame(pdf)
+    gdf = cudf.DataFrame(pdf, nan_as_null=False)
+    if request.node.callspec.id in {
```

Comment:
I'm used to seeing `pytest.param` with an xfail mark to indicate that some parametrizations will fail, but this approach seems fine too.